src/com/javabasic/lesson6/homework/Task1.java

```diff
@@ -0,0 +1,24 @@
+package com.javabasic.lesson6.homework;
+
+public class Task1 {
+    public static void main(String[] args) {
+        int min = 1;
+        int max = 100;
+
+        printMinNumber(random(min, max), random(min, max), random(min, max));
```

Comment:
first of all print your generated numbers. Because it's not clear from the output which numbers you are comparing